pyanaconda/modules/network/firewall/firewall_interface.py

```diff
@@ -114,3 +114,13 @@ def SetDisabledServices(self, disabled_services: List[Str]):
         :param disabled_services: a list of services to be enabled
         """
         self.implementation.set_disabled_services(disabled_services)
+
+    def InstallWithTask(self, sysroot: Str) -> ObjPath:
```

Comment:
Please, use the method `InstallWithTasks` that is defined for all modules.

Reply:
I didn't realize that this is a submodule of the Network module. It can stay like this.